docs/_news/news07.md

```diff
@@ -8,15 +8,15 @@ permalink: news07
 ---
 
 ## Codewind 0.7.0
-// DOUBLE CHECK -- Monday 16 December 2019
+Monday 16 December 2019
 
-#### :sparkles: Remote Deploy with Codewind: Tech Preview :sparkles:
+#### ✨ Remote Deploy with Codewind: Tech Preview ✨
 
 You can now develop your code locally and securely build and deploy your apps in the cloud! Why is this cool?
-- Free up local resources. :clap:
-- It's secure! We use [Keycloak](https://keycloak.org/) to secure the connection between your local editor and remote cloud deployment.
+- Free up local resources. 👏
+- It's secure! 🔐 We use [Keycloak](https://keycloak.org/) to secure the connection between your local editor and remote cloud deployment.
 
-// Curious? [Try it out!](link to docs) or [learn more](Link to our tech preview blog).
+Curious? [Try it out](remoteoverview.html) and keep an eye out for more information!
```

Comment:
- Maybe "Learn more" instead of "Try it out"? We don't really give instructions on the overview page.
- And I don't know if we plan to continue updating this particular page, so maybe omit "keep an eye out for more information" because it might sound like we're encouraging them to check this page again later?